packages/graphql/package.json

```diff
@@ -99,7 +99,7 @@
         "semver": "7.3.7"
```

Comment:
question (non-blocking): What about the dependencies? They were not pinned before, https://github.com/neo4j/graphql/pull/1914/files#diff-aa225a5d16977aaf2faf11389b30a7277a5fdd8a2e36d63bf0625a337bdea6b2L90-L99

Reply:
Ups, good catch, forgot to push those